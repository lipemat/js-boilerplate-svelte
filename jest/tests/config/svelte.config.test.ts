describe( 'svelte.config', () => {
	test( 'it is a module', () => {
		const svelteConfig = require( '../../../config/svelte.config' );
		expect( typeof svelteConfig ).toBe( 'object' );
		expect( svelteConfig ).toMatchSnapshot();
	} );
} );
