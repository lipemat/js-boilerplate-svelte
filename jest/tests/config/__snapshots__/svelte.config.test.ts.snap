// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`svelte.config it is a module 1`] = `
{
  "default": {
    "compilerOptions": {
      "dev": true,
    },
    "preprocess": [
      undefined,
      {
        "markup": [Function],
      },
    ],
  },
}
`;
